src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,22 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final int SIZE_OF_FIGURES_ARRAY = 6;
```

Comment:
Only one constant is enouth, because FIRST_PART_OF_ARRAY  = SIZE_OF_FIGURES_ARRAY  / 2